packages/create-sitecore-jss/src/templates/nextjs/src/Layout.tsx

```diff
@@ -33,6 +34,9 @@ const Layout = ({ layoutData }: LayoutProps): JSX.Element => {
       <Head>
         <title>{fields.pageTitle.value.toString() || 'Page'}</title>
         <link rel="icon" href={`${publicUrl}/favicon.ico`} />
+        {headLinks.map(url => (
+          <link rel="stylesheet" key={url} href={url} />
```

Comment:
"rel" shouldn't be hardcoded to "stylesheet" here, but should be a prop of the headLinks type. Instead of a string array, we should have a type with properties that represent an HTML link tag. For now, it's probably just rel and href, but we could expand on this in the future. We might need to add this type to base package (since it would be used by `getFEAASLibraryStylesheetURLs`?).